hysr/scene.py

```diff
@@ -1,13 +1,13 @@
 import typing
-
+import pam_mujoco
 from .defaults import Defaults
 
 
 class Pose:
     """
     Encapsulate a 3d position and an orientation in a format suitable
-    to Mujoco xml model file, i.e. 3d float for position and
-    string for orientation (e.g.
+    to Mujoco xml model file, i.e. 3d float for position and a 3d float for
+    orientation.
```

Comment:
Might be good to mention what kind of representation is expected for the orientation (i.e. rotation vector, euler angles, etc.).